### PR TITLE
Updated function for explore feed

### PR DESCRIPTION
**Pull request by wiktorek140**

### CODE REVIEW

File: src/api/instagram.h [hunk 1]
```diff
@@ -82,10 +85,11 @@ public slots:
...
...oid_save_multi_select,ig_android_mead,ig_android_video_single_surface,ig_android_offline_reel_feed,ig_android_video_download_logging,ig_android_last_edits,ig_android_exoplayer_4142,ig_android_snippets_haptic_feedback,ig_android_gl_drawing_marks_after_undo_backing,ig_android_mark_seen_state_on_viewed_impression,ig_android_live_backgrounded_reminder_universe,ig_android_disable_comment_public_test,ig_android_user_detail_endpoint,ig_android_location_media_count_exp_ig,ig_android_comment_tweaks_universe,ig_android_add_to_last_post,ig_save_insights,ig_android_live_enhanced_end_screen_universe,ig_android_ad_add_counter_to_logging_event,ig_android_sidecar,ig_android_direct_split_new_message_button,ig_android_grid_video_icon,ig_android_progressive_jpeg,ig_android_offline_story_stickers,ig_android_direct_inbox_unseen_hint,ig_android_top_live_titles_universe,ig_android_video_prefetch_for_connectivity_type,ig_android_sync_on_background_enhanced,ig_android_upload_reliability_use_fbupload_lib,ig_android_ad_holdout_watchandmore_universe,ig_android_samsung_app_badging,ig_android_offline_commenting,ig_android_insta_video_abr_resize,ig_android_insta_video_sound_always_on,ig_android_disable_comment";
```
Comment (neochapay) [inline src/api/instagram.h]: Maybe not comment. just remove ?

File: src/api/instagram.cpp [hunk 2]
```diff
@@ -158,7 +158,9 @@ void Instagram::profileConnect(QVariant profile)
         QJsonObject user = profile_obj["logged_in_user"].toObject();
 
         this->m_isLoggedIn = true;
+
         this->m_username_id = QString::number(user["pk"].toDouble(),'g', 10);
+
```
Comment (neochapay) [inline src/api/instagram.cpp:163]: It`s not good idea add simple line :)

Comment (wiktorek140) [reply src/api/instagram.cpp:163]: Its easier to read it sometimes :)

File: src/api/instagram.h (hunk 1 above)
Comment (wiktorek140) [reply src/api/instagram.h]: Yea, probably i forget about it

Comment (neochapay) [reply src/api/instagram.h]: Use commit --amend to fix it and i merge you code

**Review by neochapay — APPROVED** [review]

**Comment by wiktorek140** [comment]
Sorry, i'm to lazy ;)
For future i will remember this ;)
and try do it earlier ;)
Why not QUuid? because i didnt know that exist in QT ^-^

**Comment by neochapay** [comment]
I add you changes with big refactoring thanks

**Comment by wiktorek140** [comment]
It nice that is usable :)

**Pull request closed without merging**